store/models/service_agreement.go

```diff
@@ -26,13 +26,14 @@ type UnsignedServiceAgreement struct {
 
 // ServiceAgreement connects job specifications with on-chain encumbrances.
 type ServiceAgreement struct {
-	CreatedAt   Time        `json:"createdAt" storm:"index"`
-	Encumbrance Encumbrance `json:"encumbrance" storm:"inline"`
-	ID          string      `json:"id" storm:"id,unique"`
-	JobSpecID   string      `json:"jobSpecID"`
-	RequestBody string      `json:"requestBody"`
-	Signature   Signature   `json:"signature"`
-	JobSpec     JobSpec     // JobSpec is used during the initial SA creation.
+	ID            string      `json:"id" gorm:"primary_key"`
+	CreatedAt     Time        `json:"createdAt" gorm:"index"`
+	Encumbrance   Encumbrance `json:"encumbrance"`
+	EncumbranceID uint        `json:"-"`
+	RequestBody   string      `json:"requestBody"`
+	Signature     Signature   `json:"signature"`
+	JobSpec       JobSpec     // JobSpec is used during the initial SA creation.
+	JobSpecID     string      `json:"jobSpecID" gorm:"index"`
```

Comment:
Seems like the comment got cut off by the new column. Not a big deal, but would be nice to preserve, possibly make a block comment.